src/app_service/service/transaction/service.nim

```diff
@@ -423,7 +423,6 @@ QtObject:
           $route.gasAmount, gasFees, route.gasFees.eip1559Enabled, $route.gasFees.maxPriorityFeePerGas, $route.gasFees.maxFeePerGasM)
         txData.to = parseAddress(to_addr).some
         if sendType == SendType.Swap:
-          totalAmountToReceive += route.amountOut
```

Comment:
@dlipicar not sure why you added this as it was being done even if not swap above.

Reply:
ohh must've missed this when I rebased, I see we both did the exact same change in different PRs :O